.github/workflows/release.yml

```diff
@@ -49,8 +52,8 @@ jobs:
           file: ./Dockerfile
           platforms: linux/amd64
           tags: |
-            ghcr.io/fluxcd/image-reflector-controller:${{ steps.prep.outputs.VERSION }}
-            docker.io/fluxcd/image-reflector-controller:${{ steps.prep.outputs.VERSION }}
+            ghcr.io/fluxcd/${{ env.CONTROLLER }}:${{ steps.prep.outputs.VERSION }}
```

Comment:
I think you can use `${{ github.event.repository.name }}` instead of hardcoding the controller name.

Reply:
I'll try it!

Reply:
(yes it works!)